### PR TITLE
add esdl messages

### PR DESCRIPTION
**Pull request by MarkTNO**

### CODE REVIEW

File: src/grow_worker/worker.py [hunk 1]
```diff
@@ -75,10 +80,45 @@ def run_mesido(input_esdl: str) -> str:
...
+
+
+def parse_mesido_esdl_messages(
+    general_message: str, object_messages: Dict[str, str]
+) -> List[EsdlMessage]:
+    """Convert mesido messages to a list of esdl messages in omotes format.
+
+    :param general_message: general message (not related to a specific ESDL object).
+    :param object_messages: esdl object messages per object id.
+    :return: list of EsdlMessage protobuf objects.
```
Comment (cwang39403) [inline src/grow_worker/worker.py:104]: Esdl dataclass object instead of protobuf object?

File: src/grow_worker/worker.py [hunk 2]
```diff
@@ -75,10 +80,45 @@ def run_mesido(input_esdl: str) -> str:
...
+    """
+    # TODO get severity from esdl message and add list of general messages after mesido update.
```
Comment (cwang39403) [inline src/grow_worker/worker.py:106]: The mesido team does not return the severity level from their package yet, so we can only assign an ERROR level at the moment, is that right?

File: src/grow_worker/worker.py [hunk 3]
```diff
@@ -36,7 +41,7 @@ class EarlySystemExit(Exception):
     ...
 
 
-def run_mesido(input_esdl: str) -> str:
+def run_mesido(input_esdl: str) -> Tuple[Optional[str], List[EsdlMessage]]:
```
Comment (cwang39403) [inline src/grow_worker/worker.py:44]: Return type docstring (also the same for `grow_worker_task()`)

File: src/grow_worker/worker.py (hunk 1 above)
Comment (MarkTNO) [reply src/grow_worker/worker.py:104]: updated

File: src/grow_worker/worker.py (hunk 2 above)
Comment (MarkTNO) [reply src/grow_worker/worker.py:106]: yes, that will come.

File: src/grow_worker/worker.py (hunk 3 above)
Comment (MarkTNO) [reply src/grow_worker/worker.py:44]: updated

**Review by cwang39403 — APPROVED** [review]

**Review by lfse-slafleur — APPROVED** [review]
Approved with the side note that the SDK still needs to be updated once that is released.

**Pull request merged**